### ISSUE TITLE
Search (accessibility)

### ISSUE THREAD

**Issue by cmenedes** [timkeane/nyc-lib#39]
# Current Behavior
After searching an address, focus is taken to the top of the page and user cannot get past the filter tab to get to the locations using the arrow keys. There is also no indication that an address was entered successfully.

# Expected Behavior
On larger screens, the window should focus to the facility list. On smaller screens, the window should focus to the dialog that gives you the option to view the map or view the facility list. There should be an indication that an address was entered successfully.

# Failure Information (for bugs)

## Steps to Reproduce
1. Build nyc-lib as described in readme, navigate to dist/examples/ol/finderapp.html
2. After searching for an address, there should be an indication if the search was successful. Screen reader should also focus to the correct place after search (facility list on larger screens, nearest location dialog on smaller screens)
